src/onegov/org/models/organisation.py

```diff
@@ -237,3 +242,12 @@ def title_lines(self):
 
     def excluded_person_fields(self, request):
         return [] if request.is_logged_in else self.hidden_people_fields
+
+    @property
+    def event_filter_fields(self):
+        @staticmethod
+        @lru_cache(maxsize=1)
```

Comment:
Did a quick test between the suggested `cached_property` and the current implementation using `lru_cache`.
Using the `cached_property` is better as it only gets evaluated once per call on `/events`. I will switch it!

Reply:
You will need to be a little bit careful, since the `cached_property` will not be cleared automatically when `definition` changes, so a `lru_cache` might still make sense, it just needs to be defined in an outer scope.

You could also go a slightly different route and move this property to `OrgApp` and use `orm_cached`. That way the property will remain cached on the application until the organization table gets updated. There's already a couple of properties like that on `OrgApp`, such as `root_pages`.

Reply:
Good catch! I now implemented using `lru_cache` and it works!